lib/oli_web/templates/project/overview.html.eex

```diff
@@ -115,6 +118,61 @@
           <% end %>
         </div>
       </div>
+
+      <div class="row my-4">
+        <div class="col-12">
+          <h4>Delete this course</h4>
+          <button type="button" class="btn btn-danger" onclick="showDeleteModal()">Delete this Course</button>
+        </div>
+      </div>
+
     </div>
   </div>
 </div>
+<script>
+  const showDeleteModal = () => {
```

Comment:
In general, we do not want to insert any more than a line or two of raw JavaScript directly in templates. The reason being is that any JS directly in a template is not subject to ESLint checks, TypeScript compilation, minification, etc. This code here should be refactored and placed into a typescript function that gets exposed via top-level `Oli` namespace.  Take a look to see how `initActivityBridge` is exposed in `app.ts`.

Reply:
moved the function into app.ts